src/index.ml

```diff
@@ -388,10 +387,10 @@ struct
   let empty_cache = Cache.create
 
   let v ?(auto_flush_callback = fun () -> ()) ?(cache = empty_cache ())
-      ?(fresh = false) ?(readonly = false) ~log_size root =
+      ?(fresh = false) ?(readonly = false) ?(strict = false) ~log_size root =
```

Comment:
We should make sure that this parameter is respected even when the RW instance is taken from the cache (which looks to be broken currently).